packages/spaces/src/SpacesLink.js

```diff
@@ -221,6 +214,13 @@ const Link = ({
                     'pt-3': stacked,
                     'text-center': stacked,
                   })}
+                  tabIndex={0}
+                  style={{
+                    cursor: link?.url ? 'pointer' : 'not-allowed',
+                  }}
+                  {...props}
```

Comment:
I don't think every type of `SpacesLink` will necessarily have a `body`, do you know if it's safe to be spreading `props` and `rest` here?

Reply:
It is safe to spread `props`. `rest` is not though. I moved that back. I also added all the click related stuff to the children, if not passed in as a function.